$NetBSD: patch-gcc_config_x-netbsd,v 1.1 2018/05/05 04:08:36 maya Exp $

Add host hooks for NetBSD - needed for working precompiled headers.
http://gcc.gnu.org/bugzilla/show_bug.cgi?id=58397

--- /dev/null	2013-09-11 09:12:57.000000000 +0200
+++ gcc/config/x-netbsd	2013-09-10 18:57:27.000000000 +0200
@@ -0,0 +1,4 @@
+host-netbsd.o : $(srcdir)/config/host-netbsd.c $(CONFIG_H) $(SYSTEM_H) \
+  coretypes.h hosthooks.h hosthooks-def.h $(HOOKS_H)
+	$(COMPILER) -c $(ALL_COMPILERFLAGS) $(ALL_CPPFLAGS) $(INCLUDES) \
+		$(srcdir)/config/host-netbsd.c
